android-interop-testing/src/androidTest/java/io/grpc/android/integrationtest/InteropInstrumentationTest.java

```diff
@@ -110,14 +111,19 @@ private void runTest(String testCase) throws Exception {
       testCa = null;
     }
 
-    String result = null;
     try {
-      result = executor.submit(new TestCallable(
+      String result = executor.submit(new TestCallable(
               TesterOkHttpChannelBuilder.build(host, port, serverHostOverride, useTls, testCa),
               testCase)).get(TIMEOUT_SECONDS, TimeUnit.SECONDS);
       assertEquals(testCase + " failed", TestCallable.SUCCESS_MESSAGE, result);
-    } catch (ExecutionException | InterruptedException e) {
-      result = e.getMessage();
```

Comment:
Previously we were informing the test runner of the result. Why do we not want to do that any more?

Reply:
Moved the assertion outside of the try block.